src/main/java/com/google/sps/servlets/UserCardsServlet.java

```diff
@@ -49,6 +49,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       if (results != null) {
         for (Entity entity : results.asIterable()) {
+          if (entity.getProperty("status") == "DELETED") {
```

Comment:
If you were to use an enum, you'd compare to the enum itself to avoid typos.

Reply:
done